etc/schema.json

```diff
@@ -4181,63 +4220,96 @@
                     "type": "object",
                     "properties": {
                         "accepted": {
+                            "description": "Number of accepted packets",
                             "type": "integer"
                         },
                         "blocked": {
+                            "description": "Number of blocked packets",
                             "type": "integer"
                         },
                         "rejected": {
+                            "description": "Number of rejected packets",
                             "type": "integer"
                         },
                         "replaced": {
+                            "description": "Number of replaced packets",
                             "type": "integer"
                         },
                         "drop_reason": {
+                            "description": "Number of dropped packets per reason",
                             "type": "object",
                             "properties": {
                                 "decode_error": {
+                                    "description":
+                                            "Number of packets dropped due to decoding errors",
                                     "type": "integer"
                                 },
                                 "defrag_error": {
+                                    "description":
+                                            "Number of packets dropped due to defragmentation errors",
                                     "type": "integer"
                                 },
                                 "defrag_memcap": {
+                                    "description":
+                                            "Number of packets dropped due to defrag memcap exception policy",
                                     "type": "integer"
                                 },
                                 "flow_memcap": {
+                                    "description":
+                                            "Number of packets dropped due to flow memcap exception policy",
                                     "type": "integer"
                                 },
                                 "flow_drop": {
+                                    "description": "Number of packets dropped due to dropped flows",
                                     "type": "integer"
                                 },
                                 "applayer_error": {
+                                    "description":
+                                            "Number of packets dropped due to global app-layer error policy",
```

Comment:
This one is also due to exception policies, so I suggest following the same style:
"Number of packets dropped due to app-layer error exception policy"

Reply:
Thank you! :) I explicitly requested review from you bc these docs were about exception policies. Knew you'd find something. 😛